src/components/Dashboard/Contributions.js

```diff
@@ -30,7 +30,7 @@ export default () => (
             If you have questions, ask on any issue (you can tag{' '}
             <a href="https://github.com/jlengstorf">@jlengstorf</a> if you’d
             like) or hit us up{' '}
-            <a href="https://twitter.com">on Twitter at @gatsbyjs</a>.
+            <a href="https://twitter.com/gatsbyjs">on Twitter</a>.
```

Comment:
Let's keep the Twitter handle visible in the link.
```suggestion
            <a href="https://twitter.com/gatsbyjs">on Twitter at @gatsbyjs</a>.
```

Reply:
Fixed.